### DISCUSSION TITLE
Will there be a plan for multilingual support?

### DISCUSSION THREAD

**Discussion by goodmen001** [tobychui/zoraxy#288] [Ideas]
Now that you have a graphical interface, consider multilingual support for the convenience of more users

**Comment by tobychui** [comment]
@goodmen001 I guess no, unless we could find a dedicated maintainer for working on the multilingual system.

**Comment by FLUPS** [comment]
Hello,
what do you think how much time and effort will a translation / support for German language consume ?
I worked for some Android Apps as Translator and maybe have time :-)

**Reply by tobychui** [reply]
Yeah sure, that sounds great. The current issue is that I still got no idea what is the best practice to add localization without a bunch of JSON file in the code base.  Might I know what you guys use when developing multi-language support for Android Apps?
